homeassistant/components/cloud/prefs.py

```diff
@@ -419,6 +429,7 @@ def _empty_config(username: str) -> dict[str, Any]:
             PREF_CLOUD_USER: None,
             PREF_CLOUDHOOKS: {},
             PREF_ENABLE_ALEXA: True,
+            PREF_ENABLE_BACKUP_SYNC: True,
```

Comment:
This means new users will have backup sync enabled, I'm not sure if that's correct or not.